lib/src/model/messaging/ae_message.dart

```diff
@@ -0,0 +1,20 @@
+/// SPDX-License-Identifier: AGPL-3.0-or-later
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'ae_message.freezed.dart';
+part 'ae_message.g.dart';
+
+@freezed
+class AEMessage with _$AEMessage {
+  const factory AEMessage({
+    @Default('') final String address,
+    @Default('') final String genesisPublicKey,
```

Comment:
Which public is that ? Creator's genesis public Key ?

Reply:
renamed by senderGenesisPublicKey